chunky/src/res/se/llbit/chunky/ui/Chunky.fxml

```diff
@@ -151,7 +152,16 @@
           <Hyperlink fx:id="discordLink" text="Discord Server" />
           <Hyperlink fx:id="guideLink" text="jackjt8's Guide to Chunky" />
           <Separator />
-          <Label text="Chunky was created by Jesper Öqvist (jesper@llbit.se)"/>
+          <Label wrapText="true" text="Chunky was created by Jesper Öqvist (jesper@llbit.se)"/>
+          <Label wrapText="true" text="Chunky is Copyright (c) 2010-2021, Jesper Öqvist and Chunky contributors." />
+          <FlowPane>
+            <Label wrapText="true" text="Permission to modify and redistribute is granted under the terms of the " />
+            <Hyperlink fx:id="gplv3" wrapText="true" text="GPLv3 license.">
```

Comment:
We could write the full license name here. Nit: Move the `.` out of the link

Reply:
👍